profiles/common/scriptvars.lua

```diff
@@ -52,6 +52,7 @@ root.ScriptVars = {
     SW_MASK3ALPHA_OFS=2788,
     SW_MAINTHDP=3304,
     SW_RESTARTMASK=3318,
+    SW_SETREVMES_VALUE=3319,
```

Comment:
worklist has this var as 3319, would it make sense to rename it to match? @MrRevo3D

Reply:
Yeah I kinda didn't know what to do with this at first.
It gets set at initialization of the game, but doesn't seem to change from there. It's used the script in `macrosys2.scx` in all games, where its value determines whether it adds a long dash separator in the backlog (for example at the start of each chapter) using `InstSetRevMes`.
Its starting value differs between different games (`0x4000` in CCLLC PSV, `0x20000` in CC Steam and CCLCC PS4, `0x80000` in CC Android), so I figured I'd best make it a profile configuration
Frankly I think the name is terrible too, so I'm more than open to suggestions

Reply:
Renamed it and moved it to `game.lua`